libs/cart/guides/installation.md

```diff
@@ -0,0 +1,46 @@
+# Installation
+
+## Table of Contents
+
+- Package Managers
+
+  - [Installing with NPM](#installing-with-npm)
+  - [Installing with yarn](#installing-with-yarn)
+
+- Platforms
+
+  - [Daffodil's In-Memory Web API](#in-memory-web-api)
+  - [Magento](#magento)
+  - [Shopify](#shopify)
+
+## Installing with `npm`
+
+To install the cart library and its dependencies, use the following command in your terminal.
+
+```bash
+npm install @daffodil/cart @daffodil/core @ngrx/store @ngrx/effects --save
+```
+
+## Installing with `yarn`
+
+To install the cart library and its dependencies, use the following command in your terminal.
+
+```bash
+yarn add @daffodil/cart @daffodil/core @ngrx/store @ngrx/effects
+```
+
+## Post-Install
+
+After installing, an ecommerce platform driver needs to be set-up. We highly recommend installing the [In-Memory Web API](#in-memory-web-api) for fast, out-of-the-box development.
+
+## In-Memory Web API
```

Comment:
The docs below here doc feel out of place, consider moving to the driver's step.

Reply:
moved to `drivers.md`